composer.json

```diff
@@ -272,6 +274,18 @@
                 }
             }
         },
+        {
+            "type": "package",
```

Comment:
Good suggestion. I initially didn't add it because I wanted to ask the FriendsOfBehat organisation to include this, but they haven't answered my e-mails yet. I'll add it to Packagist.

Reply:
I just discussed this with @jfhovinne and he suggests to move it out of OpenEuropa because it is not fitting the philosophy. I will see if I can set up an organisation with some more open minded Behat friends :)

Reply:
The project has been moved to the new "LoversOfBehat" organisation and is published on packagist: https://packagist.org/packages/lovers-of-behat/table-extension

I will update the dependencies now.